src/modules/visit.ts

```diff
@@ -4,6 +4,7 @@ import { FetchOptions } from './fetchPage.js';
 import { ContextInitOptions } from './Context.js';
 
 export type HistoryAction = 'push' | 'replace';
+export type HistoryDirection = 'forward' | 'backward';
```

Comment:
I always use "forwards"/"backwards" 🤷‍♂️😄

Reply:
I've loosely based this on the history api which calls it `forward`. But then it again, it also calls it `back` 😵‍💫